contracts/src/v0.8/vrf/dev/ArbitrumL1Fees.sol

```diff
@@ -4,7 +4,6 @@ pragma solidity ^0.8.19;
 import {ArbGasInfo} from "../../vendor/@arbitrum/nitro-contracts/src/precompiles/ArbGasInfo.sol";
 
 /// @dev An abstract contract that provides Arbitrum specific L1 fee calculations.
-// solhint-disable-next-line contract-name-camelcase
```

Comment:
This rule no longer exists and we ignore the newer version so this line is useless